src/xcode/ENA/ENA/Source/Services/Risk/Provider/Helper/CachedAppConfiguration.swift

```diff
@@ -147,14 +147,12 @@ extension CachedAppConfiguration: AppConfigurationProviding {
 			Log.debug("fetching cached app configuration", log: .appConfig)
 			// use the cached version
 			return Just(cachedVersion)
-				.receive(on: DispatchQueue.main)
```

Comment:
Not sure we should make this change on this pressing PR, do we know if anyone is relying on this being returned on the main queue currently?

Reply:
Nope. This is not asynchronous as it `Just` returns a value unlike the other branch for fetching a config. Also, the whole Combine flow [would not work](https://forums.swift.org/t/combine-receive-on-runloop-main-loses-sent-value-how-can-i-make-it-work/28631/6) or with a lot of rework – too much for a high bug.

Reply:
I don't think the overcomplicated combine solution is necessary here, we already have working solutions that could be reused in this coordinator, but I'm not blocking here and trusting you that this works for now.